src/services/notification.service.ts

```diff
@@ -146,6 +146,55 @@ export class NotificationService {
     return;
   }
 
+  async sendVoteCount(
+    referencetype: ReferenceType,
+    referenceID: string,
+  ): Promise<void> {
+    const destination: Promise<string> = new Promise(resolve => {
```

Comment:
can we use `async await`? @RiXelanya 
need confirmation by @abdulhakim2902

Reply:
Yes, I think we can..
you can write like this
`
const destination = await (referenceType === ReferenceType.POST
  ? this.postRepository.findById().then(result => result.createdBy)
  : this.commentRepository.findById().then(result => result.userId)
)
`